MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1557,7 +1557,7 @@ private TransportStreamTimestamp GetMpegTimestamp(string path)
         {
             var packetBuffer = new byte[197];
 
-            using (var fs = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.Read))
+            using (var fs = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.Read, 1))
```

Comment:
Why not use the same `bufferSize` as the `packetBuffer` length?

Reply:
That would allocate another buffer, 1 is the magic number so `FileStream` doesn't create an internal buffer.